glossary/IIFE.md

```diff
@@ -1,3 +1,17 @@
-# IIFE
+IIFE Immediately Invoked Function Expression is a function that gets called immediately after declaration.  Is most often used to create a scoping context.
 
-IIFE section to be completed.
+An IIFE be written with the calling brackets inside of the wrapping brackets:
+```javascript
+(function foo() {
+  // [body]
+}());
+````
+
+Or with the calling brackets on the outside
+```javascript
+(function foo() {
+  // [body]
+})();
+````
```

Comment:
should all be fixed now :)

On 8 December 2015 at 00:23, Hugo Giraudel <notifications@github.com> wrote:

> In glossary/IIFE.md
> <https://github.com/HugoGiraudel/SJSJ/pull/62#discussion_r46894227>:
>
> >
> > -IIFE section to be completed.
> > +An IIFE be written with the calling brackets inside of the wrapping brackets:
> > +```javascript
> > +(function foo() {
> > +  // [body]
> > +}());
> > +````
> > +
> > +Or with the calling brackets on the outside
> > +```javascript
> > +(function foo() {
> > +  // [body]
> > +})();
> > +````
>
> One too many backtick there.; )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HugoGiraudel/SJSJ/pull/62/files#r46894227>.
>



-- 
Kind regards,

*- Robert Smith*
*Web Developer*

-----

*W*: http://rbrtsmith.com/
*T*: +46(0)73 687 5773
*E*: rbrtsmith@googlemail.com